tests/stream/test_stream_manager.py

```diff
@@ -4,25 +4,61 @@
 #  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
 #           Marisol Zeledon <marisol.zeledon@ridgerun.com>
 
+import numpy as np
 import time
 import unittest
 from unittest.mock import MagicMock
 
+from rr.actions.action_manager import ActionManager
+from rr.actions.action_manager import Trigger, TriggerError
 from rr.ai.ai_manager import AIManagerOnNewImage
 from rr.gstreamer.gst_media import GstMedia
 from rr.gstreamer.media_manager import MediaManager
-from rr.stream.stream_manager import StreamManager
 from rr.stream.stream_manager import OnNewImage
+from rr.stream.stream_manager import StreamManager
 
 
-class MockImage:
+class MockTriggerMedia:
     pass
 
 
 model = "/opt/edge_ai_apps/models/detection/TFL-OD-200-ssd-mobV1-coco-mlperf-300x300/"
 disp_width = 2040
 disp_height = 1920
-mock_image = MockImage()
+default_width = 320
+default_height = 240
+
+
+class MockImage:
+    def __init__():
+        pass
+
+    def get_mock_image():
+        return np.zeros(default_width * default_height * 3, np.uint8)
```

Comment:
return np.zeros((default_width , default_height , 3), np.uint8)

Create an image and not an array